lib/rujitsu/date.rb

```diff
@@ -0,0 +1,20 @@
+module RujitsuDate
+
+  def start_of_this_month
+    today = self.today
+    self.new(today.year, today.month)
+  end
+
+  def start_of_last_month
+    today = self.today
+    self.new(today.year, today.month) - 1.month
```

Comment:
OK - slight issue here.  ```1.month``` is a part of the ActiveSupport gem - which in turn is part of Rails.  This means it's fine for our projects but - 1 - it needs to be declared as a dependency in the gemspec and 2 - ActiveSupport is a *big* gem and a lot of non-Rails projects don't like to include it as a dependency because it's so big. 

So probably a better plan is to remove the dependency on ActiveSupport completely and use https://ruby-doc.org/stdlib-2.3.1/libdoc/date/rdoc/Date.html#method-i-3C-3C